cdc/api/v2/api_helpers.go

```diff
@@ -92,7 +92,8 @@ type APIV2Helpers interface {
 		pdClient pd.Client,
 		gcServiceID string,
 		changefeedID model.ChangeFeedID,
-		checkpointTs uint64,
+		currentCheckpointTs uint64,
+		overrideCheckpointTs uint64,
```

Comment:
Better to add  some comments to explain what is the different between `currentCheckpointTs` and `overrideCheckpointTs`.

Reply:
done